.github/scripts/python-pkg-install.sh

```diff
@@ -4,6 +4,8 @@ echo "::group::Install Python apps"
 pip3 install nose mock
 
 for app in $(ls python_apps); do
+  [[ -f "python_apps/$app/requirements.txt" ]] && pip3 install -r "python_apps/$app/requirements.txt"
```

Comment:
Please keep this as requirements-dev.txt.

The requirements file is purely for test/development dependencies that are not required for running the application. Having it be requirements.txt makes that less clear. setup.py is the canonical place for runtime dependencies